web/yo/app/scripts/services/mainutils.js

```diff
@@ -488,6 +488,14 @@ angular.module('oncokbApp')
                 return false;
             }
         }
+        function updateLastModified() {
+            $rootScope.geneTimeStamp.set('lastModifiedBy', user.name);
```

Comment:
I think changes made by developers shouldn't be included?

Reply:
If a developer runs a script and changes things systematically, the developers' information won't be written into this timestamp model in meta because this function won't be triggered.
But if a developer edits something manually, then the corresponding information should be marked in timestamp model in meta.